webtester-core/src/main/java/info/novatec/testit/webtester/pageobjects/TextField.java

```diff
@@ -44,7 +41,8 @@
     private static final String APPEND_TEXT = "changed text from '{}' to '{}' by trying to append '{}'";
 
     private static final String INPUT_TAG = "input";
-    private static final Set<String> VALID_TYPES = Sets.newHashSet("", "text", "password", "number");
+    private static final Set<String> INVALID_TYPES = Sets.newHashSet("button", "checkbox", "hidden",
+                "image", "radio", "range", "reset", "submit");
```

Comment:
I don´t like this black listing concept .. this way you define textfield as the deafult type which is not right.

I would better leave the white list way ..